.gitignore

```diff
@@ -3,4 +3,13 @@ sources/*
 !sources/README.md
```

Comment:
Is the purpose of this gitignore file to list what to ignore (and what to not ignore within some of those directories) for the entire meza installation? Does this enable someone to `git pull` and `git checkout` a new release as long as that release doesn't require a modified build from source or some other major change? Could you add some kind of introductory comment explaining this. I'm not really sure what to review in this file.